First soma point error

Second attempt to load a cell... The swc structure seems reasonable to me but I receive the following error:

```python
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-7-6a742e834617> in <module>()
      1 path_to_neuron = '../morphologies/april11s1cell-1.CNG.swc'
----> 2 neuron = nm.load_neuron(path_to_neuron)

/home/hbpschool2016/local/pythonenv/local/lib/python2.7/site-packages/neurom/io/utils.pyc in load_neuron(filename)
    101     rdw = load_data(filename)
    102     name = os.path.splitext(os.path.basename(filename))[0]
--> 103     return FstNeuron(rdw, name)
    104 
    105 

/home/hbpschool2016/local/pythonenv/local/lib/python2.7/site-packages/neurom/fst/_core.pyc in __init__(self, data_wrapper, name)
     44         neurites, sections = make_neurites(self._data)
     45         soma_check, soma_class = _SOMA_CONFIG[self._data.fmt]
---> 46         soma = make_soma(self._data.soma_points(), soma_check, soma_class)
     47         super(FstNeuron, self).__init__(soma, neurites, sections)
     48         self.name = name

/home/hbpschool2016/local/pythonenv/local/lib/python2.7/site-packages/neurom/core/_soma.pyc in make_soma(points, soma_check, soma_class)
    273         raise SomaError('Invalid soma points')
    274 
--> 275     return stype(points)

/home/hbpschool2016/local/pythonenv/local/lib/python2.7/site-packages/neurom/core/_soma.pyc in __init__(self, points)
    148             'All radii must be the same'
    149         assert np.isclose(points[0, COLS.Y] - points[1, COLS.Y], points[0, COLS.R]), \
--> 150             'The first point must be one radius below 0 on the y-plane'
    151         assert np.isclose(points[0, COLS.Y] - points[2, COLS.Y], -points[0, COLS.R]), \
    152             'The second point must be one radius above 0 on the y-plane'

AssertionError: The first point must be one radius below 0 on the y-plane
```
where the soma points are:

```
1 1 0.0 0.0 0.0 6.75281 -1
2 1 -2.29 6.34 0.0 6.75281 1
3 1 2.29 -6.34 0.0 6.75281 1
```


Morphology:
http://neuromorpho.org/dableFiles/brumberg/CNG%20version/april11s1cell-1.CNG.swc
http://neuromorpho.org/dableFiles/brumberg/CNG%20version/inv-4-24-08-s2-cell-1.CNG.swc

--- a/neurom/core/_soma.py
+++ b/neurom/core/_soma.py
@@ -87,8 +87,8 @@ class SomaCylinders(Soma):
                              /)
                             / o)
                     ______ /  )
-                (|)      ) /
-                ( o )    o )
+                (|)      ) / /
+                ( o )    o )/
                 (|)_____ )
 
       Here we have a 'side-view', with each 'o' representing a point, and the
@@ -115,7 +115,7 @@ class SomaCylinders(Soma):
                 (repr(self._points), self.center, self.radius))
 
 
-class SomaThreePointCylinders(SomaCylinders):
+class SomaNeuromorphoThreePointCylinders(SomaCylinders):
     ''' NeuroMorpho compatible soma
 
     Reference:
@@ -135,7 +135,7 @@ class SomaThreePointCylinders(SomaCylinders):
         the surface area of a sphere of radius rs.
     '''
     def __init__(self, points):
-        super(SomaThreePointCylinders, self).__init__(points)
+        super(SomaNeuromorphoThreePointCylinders, self).__init__(points)
 
         # X    Y     Z   R    P
         # xs ys      zs rs   -1
@@ -146,10 +146,11 @@ class SomaThreePointCylinders(SomaCylinders):
         assert (np.isclose(points[0, COLS.R], points[1, COLS.R]) and
                 np.isclose(points[0, COLS.R], points[2, COLS.R])), \
             'All radii must be the same'
-        assert np.isclose(points[0, COLS.Y] - points[1, COLS.Y], points[0, COLS.R]), \
-            'The first point must be one radius below 0 on the y-plane'
-        assert np.isclose(points[0, COLS.Y] - points[2, COLS.Y], -points[0, COLS.R]), \
-            'The second point must be one radius above 0 on the y-plane'
+        # These checks were turned off after https://github.com/BlueBrain/NeuroM/issues/614
+        # assert np.isclose(points[0, COLS.Y] - points[1, COLS.Y], points[0, COLS.R]), \
+        #     'The second point must be one radius below 0 on the y-plane'
+        # assert np.isclose(points[0, COLS.Y] - points[2, COLS.Y], -points[0, COLS.R]), \
+        #     'The third point must be one radius above 0 on the y-plane'
 
         r = points[0, COLS.R]
         h = points[2, COLS.Y] - points[1, COLS.Y]
@@ -157,7 +158,7 @@ class SomaThreePointCylinders(SomaCylinders):
         self.radius = math.sqrt(self.area / (4. * math.pi))
 
     def __str__(self):
-        return ('SomaThreePointCylinders(%s) <center: %s, radius: %s>' %
+        return ('SomaNeuromorphoThreePointCylinders(%s) <center: %s, radius: %s>' %
                 (repr(self._points), self.center, self.radius))
 
 
@@ -241,7 +242,7 @@ def _get_type(points, soma_class):
             # somas into their custom 'Three-point soma representation':
             #  http://neuromorpho.org/SomaFormat.html
 
-            return SomaThreePointCylinders
+            return SomaNeuromorphoThreePointCylinders
 
         return {0: None,
                 1: SomaSinglePoint}.get(npoints, SomaCylinders)

--- a/neurom/core/tests/test_soma.py
+++ b/neurom/core/tests/test_soma.py
@@ -98,8 +98,8 @@ def test_make_Soma_ThreePoint():
 
 def test_make_Soma_ThreePointCylinder():
     sm = _soma.make_soma(SOMA_THREEPOINTS_PTS, soma_class=_soma.SOMA_CYLINDER)
-    nt.ok_('SomaThreePointCylinders' in str(sm))
-    nt.ok_(isinstance(sm, _soma.SomaThreePointCylinders))
+    nt.ok_('SomaNeuromorphoThreePointCylinders' in str(sm))
+    nt.ok_(isinstance(sm, _soma.SomaNeuromorphoThreePointCylinders))
     nt.eq_(list(sm.center), [0, 0, 0])
     nt.eq_(sm.radius, 44)
 
@@ -166,7 +166,7 @@ def test_make_Soma_Cylinders():
         [0.0,   10.0, 0.0, 10.0, 1, 3,  1],
     ])
     s = _soma.make_soma(soma_3pt_neuromorpho, soma_class=_soma.SOMA_CYLINDER)
-    nt.ok_('SomaThreePointCylinders' in str(s))
+    nt.ok_('SomaNeuromorphoThreePointCylinders' in str(s))
     nt.eq_(list(s.center), [0., 0., 0.])
     nt.assert_almost_equal(s.area, 1256.6370614)
 

